### PR TITLE
Log db transaction query

### PR DESCRIPTION
**Pull request by xtremerui**
# Why is this PR needed?
when `log-db-queries` is enabled, only queries that made by db connection are logged. Those queries made by transactions are missing in the log output.

# What is this PR trying to accomplish?
Show all db queries in log output

# How does it accomplish that?
Add an intercept layer to log queries made by db transaction similar to the way it intercept db connection query.

# Reviewer Checklist
- [ ] Code reviewed
- [ ] PR acceptance performed

### CODE REVIEW

File: atc/db/log_conn.go [hunk 1]
```diff
@@ -21,21 +21,41 @@ type logConn struct {
...
+	Tx
+
+	logger lager.Logger
+}
+
+func (t *logDbTx) QueryRow(query string, args ...interface{}) squirrel.RowScanner {
+	t.logger.Debug("tx-query-row", lager.Data{"query": strip(query)})
+	return t.Tx.QueryRow(query, args...)
+}
+
```
Comment (xtreme-sameer-vohra) [inline atc/db/log_conn.go:34]: Should we also wrap the `Tx.Query` func ? There are a few uses of it in the concourse code base.

Comment (xtreme-sameer-vohra) [reply atc/db/log_conn.go:34]: Ditto for `Tx.Exec` & `Tx.QueryRowContext`

Comment (xtremerui) [reply atc/db/log_conn.go:34]: OK If there are usage then we should probably wrap them too.

Comment (xtremerui) [reply atc/db/log_conn.go]: Updated

**Pull request merged**